src/views/displaymetadata.js

```diff
@@ -820,7 +836,41 @@ function onTimer() {
   }
 }
 
-exports.showResult = function(msg, baseString, model) {
+function createDiarization() {
+  var currentSpeaker = -1;
+  var speakers = '';
+  for(var i = 0; i < tokenStartTimes.length; i++) {
+    var from = tokenStartTimes[i];
+    var tokenPerSpeaker = tokensPerSpeaker[from];
+    var token = tokenPerSpeaker.token;
+    var speaker = tokenPerSpeaker.speaker;
+    
+    if(speaker != currentSpeaker && speaker != -1) {
+      var colorClass = printf('speakerColor_%d', speaker);
+      speakers += printf("<div></div><div class='speakerInfo %s'>Speaker %d:</div>", colorClass, speaker);
+      currentSpeaker = speaker;
+    }
+    
+    if(token != '%HESITATION' && token != '~NS' && token != '~VP') {
+      speakers += token + ' ';
+    }
+  }
+
+  console.log('*** SPEAKERS ***', speakers);
```

Comment:
Hi German,
There are 3 three source files in the project, in which host value is used: app.js, models.js, and socket.js. My intention was to have only one place (I decided to use a separate file host.js) to define host value. Otherwise, it is so easy to forget to change it **all three** places. What is your idea about how to define host only once? Please be aware I do not mind to remove host.js. suggest